README.md

```diff
@@ -1,23 +1,14 @@
-# How to use this template repository
+# GNOME Calendar
 
-This is a template repository, meant to be copied to start a skeleton of other projects. This template provides a few things:
-- build workflow so that a snap is built automatically when a Pull Request (PR) is opened
-- PR template
-- README.md skeleton
+*[![gnome-calendar](https://snapcraft.io/gnome-calendar/badge.svg)](https://snapcraft.io/gnome-calendar)
+[![gnome-calendar](https://snapcraft.io/gnome-caleldar/trending.svg?name=0)](https://snapcraft.io/gnome-calendar)*
 
-A bit about the README.md below:
-- Delete everything in this "How to use this template repository" (everything above the ==== line). 
-- Replace SNAP\_NAME with the name of the snap (ex. gnome-calculator).
-- Update any section starting with "TODO" to add the info requested, and remove the TODO line.
-
-========================================================================================
-# SNAP\_NAME
+Below you will find some instructions to help you contribute to this snap. The general workflow will be to submit PRs from your fork onto the "stable" branch. Once the pull request has been *submitted*, there is a GitHub action that will automatically launch a build of the snap that you can use for testing. And then once the PR has been *merged* the GitHub action will launch a build on launchpad that will be automatically uploaded to the snap store, in the candidate channel, for more extensive testing.
 
-TODO: Add links to badges. Here's the gnome-calculator badge links for example:
-*[![gnome-calculator](https://snapcraft.io/gnome-calculator/badge.svg)](https://snapcraft.io/gnome-calculator)
-[![gnome-calculator](https://snapcraft.io/gnome-calculator/trending.svg?name=0)](https://snapcraft.io/gnome-calculator)*
+## Known bugs
 
-Below you will find some instructions to help you contribute to this snap. The general workflow will be to submit PRs from your fork onto the "stable" branch. Once the pull request has been *submitted*, there is a GitHub action that will automatically launch a build of the snap that you can use for testing. And then once the PR has been *merged* the GitHub action will launch a build on launchpad that will be automatically uploaded to the snap store, in the candidate channel, for more extensive testing.
+Snap still doesn't allow to access to Gnome Control Center through DBus, so the options to configure online
+accounts from Gnome Calendar doesn't work.
```

Comment:
Is there a bug number for this somewhere we could link to?

Reply:
The reason, IIRC, is that the accounts are configured from Gnome Control Center, so if you can't open it, you can't configure them.